GNUmakefile

```diff
@@ -28,6 +28,13 @@ USE_PYTHON_MAIN = FALSE
 
 USE_SENSEI_INSITU = FALSE
 
+USE_CONDUIT ?= FALSE
+USE_ASCENT ?= FALSE
+
+# The following must be absolute paths
+CONDUIT_DIR ?= /project/projectdirs/alpine/ghweber/ascent/uberenv_libs/spack/opt/spack/cray-cnl9-haswell/gcc-5.3.0/conduit-master-w6ilqv5kfgt2wgbftzpssjj3lttgkqkv
+ASCENT_DIR ?= /project/projectdirs/alpine/ghweber/ascent/uberenv_libs/ascent-install
```

Comment:
For `CONDUIT_DIR` and `ASCENT_DIR`, could you define them in the file `Make.WarpX` instead of GNUMakefile. 
Also, for external libraries (e.g. HDF5, FFTW, etc.), we usually ask the user to provide the path as an environment variable before typing `make`. Could you use a similar pattern for ASCENT?
(see for instance the variable `FFTW_HOME` in `Make.WarpX`)